hack/run-wsu-ci-e2e-test.sh

```diff
@@ -20,8 +20,11 @@ fi
 # The WSU playbook requires the cluster address, we parse that here using oc
 CLUSTER_ADDR=$(oc cluster-info | head -n1 | sed 's/.*\/\/api.//g'| sed 's/:.*//g')
 
+# Set up hybrid networking on the cluster, a requirement of OVNKubernetes on Windows
+oc get Network.operator.openshift.io cluster -o json | jq '.spec.defaultNetwork.ovnKubernetesConfig = {"hybridOverlayConfig":{"hybridClusterNetwork":[{"cidr":"10.132.0.0/14","hostPrefix":23}]}}' | oc apply -f -
```

Comment:
You can do `oc patch network.operator cluster --type=merge -p '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"hybridOverlayConfig":{"hybridClusterNetwork":[{"cidr":"10.132.0.0/14","hostPrefix":23}]}}}}}'` instead.

Ideally this and the `Dockerfile.tool` change should be a separate PR. However I am willing to let it slide, if you make these changes a separate commit.

Reply:
Moved to https://github.com/openshift/windows-machine-config-operator/pull/107